src/plugins/team-goal-mixin.js

```diff
@@ -28,19 +28,15 @@ export default {
 		},
 		percentageFunded() {
 			const targetAmount = this.goal?.targets?.values?.[0]?.targetLendAmount ?? 0;
-			const fundedAmount = this.goal?.participation?.values?.reduce((sum, value) => {
-				return sum + (value?.amountLent ?? 0);
-			}, 0) ?? 0;
+			const fundedAmount = this.goal?.participation?.amountLent ?? 0;
 
 			return Math.floor((fundedAmount / targetAmount) * 100) || 0;
 		},
 		participationTotalCount() {
 			return this.goal?.participation?.totalCount ?? 0;
 		},
 		fundedAmount() {
-			return this.goal?.participation?.values?.reduce((sum, value) => {
-				return sum + (value?.amountLent ?? 0);
-			}, 0) ?? 0;
+			return this.goal?.participation?.amountLent ?? 0;
```

Comment:
`amountLent` is a string, but fundedAmount is expected to be a number by the components that use it. This should be converted to a number here to avoid unexpected results.